src/services/users/index.js

```diff
@@ -123,10 +132,45 @@ const inviteUser = async (email) => {
 	return signupInfo
 }
 
+const getNewTokens = async (email, currentRefreshToken) => {
+	const result = await  db.query(
+		`SELECT * FROM users 
+		WHERE email = $1 
+		AND "refreshToken" = $2
+		`,
+		[email, currentRefreshToken]
+	)
+	const userProfile = result.rows[0]
+
+	if (!userProfile) {
+		throw customError(refreshTokenIsInvalidError)
+	}
+
+	const user = userProfile
+
+	const body = { id: user.id, email: user.email }
+	const accessToken = await generateAccessToken({
+		data: {user : body}, 
+		expiry : '15m'
+	})
+	
+	const refreshToken = await generateRefreshToken()
+	 await db.query(
```

Comment:
@beblicarl, this query appears twice now, you can extract this a function which takes the refreshToken and id and call it in both places

Reply:
yes please